### PR TITLE
Create crop-for-coins.md

### PR DESCRIPTION
**Pull request by minoyahia**
## Information

- [ ] This PR fixes a formatting issue.
- [ ] This PR corrects the information in an existing guide.
- [x] This PR adds a new guide.

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] I am sure the guide is formatted properly.

## Licensing

- [x] I agree that my contributions shall be licensed under the [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/),
  and that SkyKings may distribute my contributions under the same license on their website.

### CODE REVIEW

File: guides/crop-for-coins.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  Learn what crop makes the most coins per hour
+category: Money
+author: MohIsDaddy
+tags:
+  - skyblock money making method
+```
+With all the money making methods out there, farming has to be one of the greatest
+of them, being almost fully afk-able and making upwards to 10m per hour and almost 
+16m an hour while being maxed, so you might be asking, what crop should I be farming
+to make the most? well then, you came to the right place!
```
Comment (plun1331) [inline guides/crop-for-coins.md]:
```suggestion
to make the most? Well then, you came to the right place!
```

File: guides/crop-for-coins.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Before we start you must keep in mind that a mooshroom cow is always better for coins
+over elephant, but you'll be going to loose quite the amount of collection.
```
Comment (plun1331) [inline guides/crop-for-coins.md:16]: what

File: guides/crop-for-coins.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+While using bountiful with a decent setup and a melon dicer, you'll be going to be
+getting the most amount of coins, ranking as best with a potentail 16m per hour
```
Comment (plun1331) [inline guides/crop-for-coins.md]:
```suggestion
While using bountiful with a decent setup and a melon dicer, you'll be 
getting the most amount of coins, ranking as best with a potential 16m per hour
```

**Review by plun1331 — CHANGES_REQUESTED** [review]

File: guides/crop-for-coins.md (hunk 2 above)
Comment (minoyahia) [reply guides/crop-for-coins.md:16]: yes

**Review by minoyahia — COMMENTED** [review]
smash

**Pull request merged**